mattermost-plugin/server/plugin.go

```diff
@@ -98,8 +98,13 @@ func (p *Plugin) OnActivate() error {
 		return fmt.Errorf("error initializing the DB: %v", err)
 	}
 
+	baseURL := ""
```

Comment:
I think it should be `"."` instead of `""` right?

Reply:
Thought about it but if the user is using a subpath, they should have set `SiteURL`. In any case it is more correct, so happy to make the change :)

Reply:
Yes, lets do the change, for correctness and we can merge it.

Reply:
Fixed in cceb3801679b65bb14336e91909ea4b02f448903

Reply:
Actually it looks like the `ServerRoot` doesn't work with a relative URL

Reply:
@mgdelacroix we can also use this - https://developers.mattermost.com/extend/plugins/manifest-reference/#required_configuration to prevent the plugin from starting without a site URL specified. Would this make things simpler?

Reply:
Thanks for the pointer @harshilsharma63!! We could use that, but if we can support the empty `SiteURL` reliably I think it is preferable.

@jespino found the issue. The URL cannot be relative as we're running the web interface in a subpath already, so it fails when filling the `index.html` template with the tag `<script defer src="./plugins/focalboard/static/main.js"></script>` and load it from the subpath, it's actually intended to be absolute.